src/features/lessons/edit-lesson/edit-lesson.js

```diff
@@ -1,20 +1,20 @@
 import React, { useCallback, useEffect, useState } from 'react';
 import { useHistory, useParams } from 'react-router-dom';
 import { shallowEqual, useSelector } from 'react-redux';
-import { editLessonSelector, studentsSelector, loadStudentGroupsSelector, lessonsSelector,
+import {
+  editLessonSelector, studentsSelector, loadStudentGroupsSelector, lessonsSelector,
   fetchLessons, globalLoadStudentGroups, loadStudents, editLesson,
 } from '@/models';
 import { useActions, paths } from '@/shared';
 
 import { WithLoading } from '@/components';
-import { lessonValidation } from "@features/validation/validation-helpers";
+import { editLessonValidation, StudentsFormDataValidation } from '@features/validation/validation-helpers';
```

Comment:
Please, change to one code-style here and in validation-helpers
StudentsFormDataValidation -> studentsFormDataValidation